RevenueCatUI/CustomerCenter/Data/CustomerCenterData.swift

```diff
@@ -0,0 +1,93 @@
+//
+// CustomerCenterData.swift
+//  
+//
+//  Created by Cesar de la Vega on 28/5/24.
+//
+
+import Foundation
+
+struct CustomerCenterData: Decodable {
+
+    let id: String
+    let paths: [HelpPath]
+    let title: LocalizedString
+    let supportEmail: String
+    let appearance: Appearance
+
+    enum HelpPathType: String, Decodable {
+        case missingPurchase = "MISSING_PURCHASE"
+        case refundRequest = "REFUND_REQUEST"
+        case changePlans = "CHANGE_PLANS"
+        case cancel = "CANCEL"
+        case unknown
+    }
+
+    struct HelpPath: Decodable {
+
+        let id: String
+        let title: LocalizedString
+        let type: HelpPathType
+        let promotionalOffer: PromotionalOffer?
+        let feedbackSurvey: FeedbackSurvey?
+
+    }
+
+    struct LocalizedString: Decodable {
+        
+        // swiftlint:disable:next identifier_name
+        let en_US: String
```

Comment:
leaving this for now since this is most likely going to change. We will probably be passing the locale as a parameter and the API will only return stuff in that locale